<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="USVAWXMCS0681" name="com.qa.viacom.tests.OrangeHRMTest" tests="2" failures="2" timestamp="19 Feb 2022 05:39:40 GMT" time="35.076" errors="0">
  <testcase name="loginLogoutTest" time="17.026" classname="com.qa.viacom.tests.OrangeHRMTest">
    <failure type="org.junit.ComparisonFailure" message="[Title is not mathing] expected:&lt;&quot;[OrangeHRM]&quot;&gt; but was:&lt;&quot;[Login - PHPTRAVELS]&quot;&gt;">
      <![CDATA[org.junit.ComparisonFailure: [Title is not mathing] expected:<"[OrangeHRM]"> but was:<"[Login - PHPTRAVELS]">
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at com.qa.viacom.tests.OrangeHRMTest.loginLogoutTest(OrangeHRMTest.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- loginLogoutTest -->
  <testcase name="loginLogoutTest" time="18.050" classname="com.qa.viacom.tests.OrangeHRMTest">
    <failure type="org.junit.ComparisonFailure" message="[Title is not mathing] expected:&lt;&quot;[OrangeHRM]&quot;&gt; but was:&lt;&quot;[Login - PHPTRAVELS]&quot;&gt;">
      <![CDATA[org.junit.ComparisonFailure: [Title is not mathing] expected:<"[OrangeHRM]"> but was:<"[Login - PHPTRAVELS]">
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at com.qa.viacom.tests.OrangeHRMTest.loginLogoutTest(OrangeHRMTest.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- loginLogoutTest -->
</testsuite> <!-- com.qa.viacom.tests.OrangeHRMTest -->
